js/hvacController.js

```diff
@@ -559,44 +574,37 @@ hvacController.prototype.initButtons = function () {
 			hvacController.prototype.status.ACComand = true;
 
 			hvacController.prototype.onFanChanged(true);
-			//carIndicator.setStatus("Fan", true);
-			//carIndicator.setStatus("ACCommand", true);
 
-			//carIndicator.setStatus("airRecirculation", false);
-			//carIndicator.setStatus("RecircReq", 0);
 			hvacController.prototype.status.airRecirculation = false;
 			hvacController.prototype.status.RecircReq = 0;
 
 			if (autoACStatus.targetTemperatureRight < 16 || autoACStatus.targetTemperatureRight > 28) {
+				hvacController.prototype.status.targetTemperatureRight = 22;
 				hvacController.prototype.onTargetTemperatureRightChanged(22);
-
-				//carIndicator.setStatus("targetTemperatureRight", 22);
-				//carIndicator.setStatus("FrontTSetRightCmd", 22);
 			}
 			if (autoACStatus.targetTemperatureLeft < 16 || autoACStatus.targetTemperatureLeft > 28) {
 				hvacController.prototype.onTargetTemperatureLeftChanged(22);
-				//carIndicator.setStatus("targetTemperatureLeft", 22);
+				hvacController.prototype.status.targetTemperatureLeft = 22;
 				//carIndicator.setStatus("FrontTSetLeftCmd", 22);
 			}
 		} else {
 			$("#fan_control_auto").removeClass("on");
 
-   		    //carIndicator.setStatus("fanSpeed", autoACStatus.fanSpeed);
+   		    hvacController.prototype.status.fanSpeed = autoACStatus.fanSpeed;
 		    //carIndicator.setStatus("FrontBlwrSpeedCmd", autoACStatus.fanSpeed);
```

Comment:
Code cleanup looks good. Do the //carIndicator lines need to stay further down as you've removed most of them? If I  had full comprehension of those commented out lines would I understand them without further comment?